### ISSUE TITLE
Reports webapp with gunicorn

### ISSUE THREAD

**Issue by microfuge** [galaxyproject/ansible-galaxy#195]
Hi,

Please let me know if I am wrong, but the "Galaxy reports" app will not run with uwsgi as now Galaxy uses Gunicorn. The templates still mentions uwsgi https://github.com/galaxyproject/ansible-galaxy/blob/main/templates/systemd/galaxy-reports.service.j2 

And update to the role for this would be great. Or any advice on running the Reports app with gunicorn would also be very helpful.

Thanks!

**Comment by hexylena** [comment]
@adRn-s sure it's valid, but it's not used by Galaxy any more. Configuration for reports is now solely through gunicorn + gravity.

@microfuge we should definitely support this going forward, but I'm not sure when someone will update this issue. 
@natefoo has the most experience with gravity which is now a necessary part of galaxy configuration.

**Comment by mvdbeek** [comment]
> @natefoo has the most experience with gravity which is now a necessary part of galaxy configuration.

No, you can use whatever you want, but it's certainly a lot easier to use gravity.

**Comment by hexylena** [comment]
@mvdbeek if it were added to this role, we would want to use gravity, wouldn't want to special case reports.
